Rater.Domain/DataTransferObjects/MetricDto/MetricOfParticipantWinnerDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Rater.Domain.DataTransferObjects.MetricDto
+{
+    public class MetricOfParticipantWinnerDto
```

Comment:
This is not a good class name it shouldn't contain words like `of`

Reply:
And you don't need to make everything class

Reply:
We're trying to work in Object Oriented methodology not lexical/verbal talking

Reply:
Yes Resulting needs token you can check SpaceController